### PR TITLE
audio

### PR DESCRIPTION
**Pull request by xvii-tommy**
latest for code review.
Still working on audio and token parsing but have found the resources you recommended.

### CODE REVIEW

**Comment by Adrir** [comment]
Please remove the unnecessary folder and consolidate your code into one (or fewer) files.

**Comment by Adrir** [comment]
Is this the latest version?

File: code for tinkering audio/audio_Envelops_And_Echos.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+def echoes(delay, num):                                             
+  sound = makeSound(pickAFile())
+  s1 = getLength(sound)                                              #make empty sound
+  s2 = s1 + (delay * num)                                            
+  canvas = makeEmptySound(s2)                                        #define sound length
+  
+  echoAmplitude = 1.0
+  for echoCount in range(1, num):                                   
+    echoAmplitude = echoAmplitude * 0.5                             #volume of echo is halfed each time
+    for posns1 in range(0, s1):
```
Comment (Adrir) [inline code for tinkering audio/audio_Envelops_And_Echos.py:12]: Please do not use numbers at the end of variable and/or function names.

File: code for tinkering audio/audio_Splice_And_Swap.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#function for splicing and swapping audio.
+
+def spliceAndSwap():
+  s1 = makeSound(pickAFile())                           
+  s2 = makeEmptySoundBySeconds(10)                     #This creates a blank canvas
+  targetIndex = 1
+  for sourceIndex in range (20000, 40000):             #gets the sample values between the ranges
```
Comment (Adrir) [inline code for tinkering audio/audio_Splice_And_Swap.py:7]: Where have the literals 20000 and 40000 come from?

File: code for tinkering audio/tone_generation.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#function for generating a tone.
+
+def toneGeneration(sec, freq, amplitude):
+    buildSin = makeEmptySoundBySeconds(secs)                  #building the clip
+    sampleRate = getSamplingRate(buildSin)
```
Comment (Adrir) [inline code for tinkering audio/tone_generation.py:5]: [Naming Conventions](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables). Should be: sample_rate.

**Comment by xvii-tommy** [comment]
Hi, yes this is the latest version I still have token parsing and random
audio to do.

**Comment by Adrir** [comment]
Make a pull request when these have been added.

**Pull request closed without merging**